api/src/main/java/com/xwiki/googleapps/internal/CookieAuthenticationPersistence.java

```diff
@@ -17,50 +17,39 @@
  * Software Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA
  * 02110-1301 USA, or see the FSF site: http://www.fsf.org.
  */
-package org.xwiki.apps.googleapps;
+package com.xwiki.googleapps.internal;
 
 import org.xwiki.component.annotation.Role;
 
-import com.xpn.xwiki.XWikiContext;
-import com.xpn.xwiki.XWikiException;
-
 /**
  * Set of methods for the management of the cookies.
  *
- * @since 3.0
  * @version $Id$
+ * @since 3.0
  */
 @Role
-public interface CookieAuthenticationPersistence
+interface CookieAuthenticationPersistence
```

Comment:
It can remain public from Java point of view even if it's an internal API. Do you need to restrict the access to the package level?

Reply:
Well, this interface is not meant to be used by others so I prefer to make it internal.
We need an interface so as to use components.

Reply:
See discussion about the components below. This is solved.